drift/test/database/statements/order_by_test.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:drift/drift.dart' hide isNull;
+import 'package:mockito/mockito.dart';
+import 'package:test/test.dart';
+
+import '../../generated/todos.dart';
+import '../../test_utils/test_utils.dart';
+
+void main() {
+  late TodoDb db;
+  late MockExecutor executor;
+
+  setUp(() {
+    executor = MockExecutor();
+    db = TodoDb(executor);
+  });
+
+  test('when nullsOrder is null it ignored', () async {
+    final query = db.select(db.users);
+    query.orderBy([(tbl) => OrderingTerm(expression: tbl.name)]);
+    await query.get();
+    verify(executor.runSelect(
```

Comment:
Thanks for these tests! By the way, if you only wanted to test for the SQL generated by an `OrderingTerm`, you could also do `expect(OrderingTerm(expression: tbl.name), generates('name ASC'))`. But doing it like this is fine too, it's always good to test the thing in a full and working context :)